WooCommerce/Classes/Spotlight/SearchableActivityConvertable.swift

```diff
@@ -0,0 +1,58 @@
+import CoreSpotlight
+import UIKit
+import MobileCoreServices
+
+/// Custom NSUSerActivity types for the Woo app. Primarily used for navigation points.
+///
+enum WooActivityType: String {
+    case dashboard               = "com.automattic.woocommerce.dashboard"
+    case orders                  = "com.automattic.woocommerce.orders"
+    case products                = "com.automattic.woocommerce.products"
+    case payments                = "com.automattic.woocommerce.payments"
+}
+
+@objc protocol SearchableActivityConvertable {
```

Comment:
I'll let a native speaker confirm this: Should it be "Convertible" (with `i`) ? I did some quick Google search and found both results that Convertible can be a synonym, as well as that is incorrect spelling. Maybe changes depending on the type of English used? 🤔 

For what is worth, in other instances of this word through the app we use the `i` variant.

Reply:
Apple uses [convertible](https://developer.apple.com/documentation/swift/customstringconvertible), so we have a good reason to change it :) Done in 739f5a169b6c3607cfa1f7da99e5579458cd5f39